### DISCUSSION TITLE
Discrepancy between definitions of “Allowed Amount”

### DISCUSSION THREAD

**Discussion by nicholas-dahl-CTS** [CMSgov/price-transparency-guide#72] [Q&A] [answered]
There seems to be a discrepancy in how the final rule defines the allowed amount to be reported in the Allowed Amounts file.  Specifically, the rule text states:

"When disclosing an out-of-network allowed amount under this requirement, the Departments proposed to require a plan or issuer to disclose the actual amount the plan or issuer paid to the out-of-network provider, plus the participant’s, beneficiary’s, or enrollee’s share of the cost. For instance, if the out-of-network allowed amount for a covered service was $100, and the plan or issuer paid 80 percent of the out-of-network allowed amount ($80) per the terms of the plan or coverage, so that the participant, beneficiary, or enrollee was responsible for paying twenty percent of the out-of-network allowed amount ($20), the plan or issuer would report an out-of-network allowed amount of $100. This unique payment amount would be associated with the particular covered item or service (identified by billing code) and the particular out-of-network provider who furnished the item or service (identified by NPI, TIN, and Place of Service Code)."

So, the rule states that plans are required to report the actual amount paid by the plan, but go on to state that in the above example that would be $100 (instead of $80 which is what would have actually been paid by the plan).  The schema does not clarify further, as it also states “actual amount paid by the plan”.  Can we get additional clarification / guidance on how this amount should be reported?  Thanks!

**Comment by clovis517** [comment, marked as answer]
_So, the rule states that plans are required to report the actual amount paid by the plan,_ 
Both the Final Rule and the Out-Of-Network Payment Object follow the above language with ... **plus the participant’s, beneficiary’s, or enrollee’s share of the cost.**
so I don't see the discrepancy.   The $20 in the example would be copay, coinsurance, a 20% out of network sanction, or deductible.  Note that CMS is not describing balance billing as part of the "beneficiary’s, or enrollee’s share of the cost", otherwise the requirement would be described as the Provider Billed Charge amount.
